pkg/rbac/const.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2020 Clastix Labs.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rbac
+
+import (
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	provisionerRoleName = "capsule-namespace:provisioner"
```

Comment:
I guess we need to export the `deleterRoleName` constants since we need it here:
https://github.com/clastix/capsule/blob/0f935d53b7acb17379e749a2d64e2f631adb6779/controllers/tenant_controller.go#L510-L521

Or rather, we could move that piece of code under this package: what do you think?

Reply:
I will make the exportable :)